### PR TITLE
New country rules for CZE, IDN and SVK

### PR DESCRIPTION
**Pull request by sheilagomes**
Add Czech Republic ('CZE'), Indonesia ('IDN') and Slovakia ('SVK') country rules. Tracked by tasks [LOC-8769](https://vtex-dev.atlassian.net/browse/LOC-8769), [LOC-8800](https://vtex-dev.atlassian.net/browse/LOC-8800).

### CODE REVIEW

File: src/script/countries/CZE.coffee [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		return phone
+
+	splitNumber: (number) =>
+		if number.length is 9
+			return Phone.compact number.split(/(\d{3})(\d{3})(\d{3})/)
+
+		return [number]
+
+# register
+czechrepublic = new Czechrepublic()
```
Comment (lucasecdb) [inline src/script/countries/CZE.coffee:31]: the class name is in PascalCase

```suggestion
czechrepublic = new CzechRepublic()
```

File: src/script/countries/CZE.coffee [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		@nationalDestinationCode =
+			["2","31","32","35","37","38","39","41","47","46","48","49","51","53","54","55","59","56","57","58"]
+	
+	specialRules: (withoutCountryCode, withoutNDC, ndc) =>
+	phone = new PhoneNumber(@countryNameAbbr, @countryCode, ndc, withoutNDC)
+
+	if withoutNDC.length is 6
+		if ndc.length is 3 and ndc[0] is '6' or ndc[0] is '7'
+			phone.isMobile = true
+		return phone
```
Comment (lucasecdb) [inline src/script/countries/CZE.coffee:22]: missing an indentation level in the method body

```suggestion
	specialRules: (withoutCountryCode, withoutNDC, ndc) =>
		phone = new PhoneNumber(@countryNameAbbr, @countryCode, ndc, withoutNDC)

		if withoutNDC.length is 6
			if ndc.length is 3 and ndc[0] is '6' or ndc[0] is '7'
				phone.isMobile = true
			return phone
```

**Review by wdsrocha — APPROVED** [review]
I see that all tests are passing, so it LGTM ✨

**Comment by lucasecdb** [comment]
@kaio-donadelli could you please do the australia and new zealand additions in another PR? the rules for Indonesia, Czech Republic and Slovakia were already released in a new version and so we should merge this PR ASAP in the `main` branch. You can continue your work on another branch/PR, ok?

**Comment by kaio-donadelli** [comment]
> @kaio-donadelli could you please do the australia and new zealand additions in another PR? the rules for Indonesia, Czech Republic and Slovakia were already released in a new version and so we should merge this PR ASAP in the `main` branch. You can continue your work on another branch/PR, ok?

I've just reverted my commits and put them in another branch.

**Pull request merged**